cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -76,23 +77,23 @@ receivers:
       - job_name: 'otel-collector'
         scrape_interval: 10s
         static_configs:
-        - targets: ['0.0.0.0:8888']
+        - targets: ["${SPLUNK_LISTEN_INTERFACE}:8888"]
```

Comment:
This is the address to query, so if we want to influence the internal prom server address we should specify it in `service::telemetry::metrics::address`

Reply:
and I wonder if we should hardcode to 127.0.0.1, instead of tying this up to this env var.

Reply:
That might be a separate PR that I could land first.

Reply:
I'm wrong, appears the default behavior w/ `:8888` resolves to `:::8888` so this seems ok as is though an additional env var may be warranted in the future.

Reply:
Right, if you don't mind, this might be later instead of landing it first.